### PR TITLE
Update kalibro_client

### PR DESCRIPTION
**Pull request by rafamanzo**
This update is necessary to run the newest service versions.

### CODE REVIEW

**Comment by diegoamc** [comment]
Do we have a clue on the reason why some many tests are failing?

File: app/helpers/kalibro_modules_helper.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def sort_by_granularity_and_name(module_results)
       if (a.kalibro_module.granularity == b.kalibro_module.granularity)
         a.kalibro_module.name <=> b.kalibro_module.name
       else
-        (ComparableGranularity.new(b.kalibro_module.granularity.to_sym) <=> ComparableGranularity.new(a.kalibro_module.granularity.to_sym))
+        (ComparableGranularity.new(b.kalibro_module.granularity.type) <=> ComparableGranularity.new(a.kalibro_module.granularity.type))
```
Comment (diegoamc) [inline app/helpers/kalibro_modules_helper.rb]: Don't we need to convert the type into a symbol?

Comment (rafamanzo) [inline app/helpers/kalibro_modules_helper.rb]: `Granularity#to_sym` already returns a symbol.

**Comment by diegoamc** [comment]
Nice job! :100:

**Pull request merged**